config.yaml

```diff
@@ -44,6 +44,10 @@ options:
       Comma separated list of IP address CIDR ranges that should be allowed for
       federation, identity servers, push servers, and for checking key validity
       for third-party invite events.
+  limit_remote_rooms_complexity:
+    type: float
+    description: if set, the room "complexity" will be checked before a user
+      joins a new remote room.
```

Comment:
Might be good to mention the user will be unable to join in the description.

Reply:
Good point, changed.